HW13/script.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+tput setaf 5; echo "Hello. This is a script for comparing three numbers."
+echo "Enter exit(e) to exit"; tput setaf 7 
+
+while :; do
+tput setaf 4; echo "Enter the number A"; tput setaf 7 
+read A
+if [[ $A == "exit" ]] || [[ $A == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+tput setaf 4; echo "Enter the number B"; tput setaf 7 
+read B
+if [[ $B == "exit" ]] || [[ $B == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+tput setaf 4; echo "Enter the number C"; tput setaf 7 
+read C
+if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+if [ -z $A ] || [ -z $B ] || [ -z $C ]; then
+    tput setaf 2; echo "The value A, B or C is empty"; tput setaf 7
+fi
+
+if [[ $A == "Zero" ]] || [[ $A == "zero" ]]; then
```

Comment:
Too many code for letters. use `declare` instead

Reply:
Done!
But I didn't find a way to add a character check when using `declare`